packages/python/plotly/plotly/tests/test_optional/test_px/test_px_input.py

```diff
@@ -233,6 +246,24 @@ def test_build_df_with_index():
     assert_frame_equal(tips.reset_index()[out["data_frame"].columns], out["data_frame"])
 
 
+def test_build_df_using_interchange_protocol_mock(add_interchange_module_for_old_pandas):
```

Comment:
Using a mock test, "plotly" do not need to have a test for every library that supports this protocol.